packages/website/dummy-data/complete-config-project/docs.config.js

```diff
@@ -34,6 +36,8 @@ module.exports = () => ({
     path.join(__dirname, 'otherpackages', '*'),
   ],
   packagesDescription: 'A custom description for the packages section',
+  packagesImgSrc: '/static/simplify.svg',
```

Comment:
if we have the opportunity, i think it would be good to combine the related config into top-level objects.
```js
packages: {
  paths: [...],
  description: '',
  imgSrc: '',
}
```

Reply:
👍 This makes sense. @dpisani-atl is moving config around atm, but I think this would be a good backlog ticket.

Reply:
created https://jdog.jira-dev.com/browse/AFP-368